paper.tex

```diff
@@ -291,17 +291,43 @@ \subsubsection{A Scalable Keys and Data Entry Points Management}
 
 %data trace
 \subsubsection{A Traceable Data Stream}
-Data tracability is an important security issue that allows users to track the changes of data. Currently, hash is widely used to generate checksums of entire package of files, users can compare the hash value of files in order to check the integrity. Yet the hash values of each new version are unrelated and it's hard to point out the differences between versions. MAM benefits from the singly linked-list data structure which attaches messages chronologically, users can easily track the footprints of data change logs as well as checking the validity of modifications with the signature.
+Data traceability is an important security issue that allows users to track the changes of data. Currently, hash is widely used to generate checksums of entire package of files, users can compare the hash value of files in order to check the integrity. Yet the hash values of each new version are unrelated and it's hard to point out the differences between versions. MAM benefits from the singly linked-list data structure which attaches messages chronologically, users can easily track the footprints of data change logs as well as checking the validity of modifications with the signature.
 
 \subsection{Delegate MAM operations to Tangle-accelerator}
 \label{section:ta_endpoint}
-MAM builds a secure and authenticated communication protocol on top of multiple cyptosystems, which low-level devices need to spend a lot of resources to apply it, and they may not even support built-in libraries in regular operating system. To overcome these concerns, we propose another approach that allows data providers to delegate the MAM operations to Tangle-accelerators, proxy servers with high computation power, which can accelerates the interactions with Tangle while ensuring the privacy of data providers.
+MAM builds a secure and authenticated communication protocol on top of multiple cryptosystems, which low-level devices need to spend a lot of resources to apply it, and they may not even support built-in libraries in regular operating system. To overcome these concerns, we propose another approach that allows data providers to delegate the MAM operations to Tangle-accelerators, proxy servers with high computation power, which can accelerates the interactions with Tangle while ensuring the privacy of data providers.
 
-\subsubsection{Plight for Low-level edge device}
+\subsubsection{Plight for Low-level Edge Device}
+MAM requires quite huge computational power which causes a burden for low-level devices to adopt MAM service. Therefore we introduce a proxy server, tangle-accelerator, which performs the heavy MAM computations, so low-level devices can be relieved from the enormous computation.
```

Comment:
This paragraph is actually similar to the previous one.
Can you make them into one paragraph?
Then maybe we can also remove the subsubsection `Plight...` ... :thinking: